### ISSUE TITLE
Add unit tests to arrival time algorithms

### ISSUE THREAD

**Issue by liamphmurphy** [codeforpdx/opentransit-metrics#13]
(Original source: https://github.com/trynmaps/metrics-mvp/issues/403)

**Original Description**:
Add tests to "data science" functions to allow newcomers to get better grasp of what functions are doing, shape of data input/output, and to allow easier changing/fixing of functions.

Functions to test are in:
backend/models/eclipses.py
backend/models/wait_times.py
backend/models/trip_times.py

Any questions about the functions or data should be directed to @youngj (Jesse Young on Slack).

**For Our Purposes**:
The above is pretty accurate for what would be useful to us. We can explore other places for unit tests if needed, but this is a good start.

**Comment by youngj** [comment]
There are some unit tests in https://github.com/codeforpdx/opentransit-metrics/tree/master/backend/tests although none for eclipses.py yet. These unit tests are run automatically via GitHub Actions - e.g. https://github.com/codeforpdx/opentransit-metrics/runs/5402144883?check_suite_focus=true .

**Comment by liamphmurphy** [comment]
Good catch, I'll edit the issue to add tests for eclipses.py if that makes sense?
